package.json

```diff
@@ -38,15 +38,14 @@
     "eslint-config-crowdstrike": "10.1.0",
     "eslint-config-crowdstrike-node": "link:",
     "eslint-plugin-json-files": "^4.0.0",
-    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-n": "^14.0.0",
     "remark-cli": "^12.0.0",
     "remark-preset-lint-crowdstrike": "^4.0.0",
     "renovate-config-standard": "^2.0.0",
     "standard-version": "^9.0.0"
   },
   "peerDependencies": {
     "eslint-config-crowdstrike": ">=1",
-    "eslint-plugin-json-files": ">=0.8",
-    "eslint-plugin-node": ">=10"
```

Comment:
Why wouldn't this be "eslint-plugin-n": ">=14"?

Reply:
You're 100% right